PIL/ImageGrab.py

```diff
@@ -51,11 +51,31 @@ def grab(bbox=None):
 
 def grabclipboard():
     if sys.platform == "darwin":
-        raise NotImplementedError("Method is not implemented on OS X")
-    debug = 0  # temporary interface
-    data = Image.core.grabclipboard(debug)
-    if isinstance(data, bytes):
-        from PIL import BmpImagePlugin
-        import io
-        return BmpImagePlugin.DibImageFile(io.BytesIO(data))
-    return data
+        f, file = tempfile.mkstemp('.jpg')
```

Comment:
Don't use builtin `file`, and maybe use more descriptive name than `f`.

Perhaps `file` -> `filename` and `f` -> `fh`.